hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -54,19 +54,22 @@
         "item_type": "file.photoshop",
     },
     "Rendered Image": {
-        "extensions": ["dpx", "exr"],
+        "extensions": ["dpx", "exr", "cin"],
         "icon": "image_sequence.png",
         "item_type": "file.image",
     },
     "Texture Image": {
-        "extensions": ["tiff", "tx", "tga", "dds", "rat"],
+        "extensions": ["tiff", "tx", "tga", "dds", "rat", "cin"],
         "icon": "texture.png",
         "item_type": "file.texture",
     },
+    "Reference Image": {
+        "extensions": ["jpg", "jpeg"],
+        "icon": "texture.png",
+        "item_type": "file.image",
```

Comment:
This is ok, but would suggest considering something like `file.ref`. That way, plugins can have item filters like `[file.*]` or `[file.image, file.ref]` to act on both types. But you could also have plugins that only act on ref images by using `[file.ref]` for the filter. Just a suggestion.